### PR TITLE
Initial version for AboutPage.

### PR DESCRIPTION
**Pull request by artruusi**
This is related to issue #22. Initial version where licence texts should be added later

### CODE REVIEW

File: src/components/AboutPageView.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        <Text style={styles.title}>{ABOUT_TITLE}</Text>
+        <Text style={styles.text}>{ABOUT_TEXT1}</Text>
+      </View>
+    );
+  }
+}
+
+const mapStateToProps = (state: RootReducer) => ({
+  isAuthenticated: state.auth.isAuthenticated
+});
```
Comment (Voya100) [inline src/components/AboutPageView.tsx]: Is this needed anywhere?

Comment (artruusi) [reply src/components/AboutPageView.tsx]: Yes and no. isAuthenticated is not used but rest is required for it to work. Should we check if user is authenticated or not? Do we care and is it even possible to not be and end up in this view?

Comment (Voya100) [reply src/components/AboutPageView.tsx]: Probably not necessary to check. The view doesn't need to fetch anything from the backend, so there is no interest in knowing whether user is currently logged in or not (user must have been at least before if they have succeeded to navigate here).

This view probably doesn't need any Redux functionality, so all connect related things could be removed if desired.

Comment (artruusi) [reply src/components/AboutPageView.tsx]: I removed them.

**Pull request merged**